_commands/basics/kill.md

```diff
@@ -3,22 +3,31 @@
 
 kill
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
 
-<!-- minimal example -->
+`kill` is used to send a signal to a process.
+
 ~~~ bash
-*short* example of command usage and output
+kill [-s] [-l] %pid
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
 
-##### Break it down
+#### `kill -9 -1`
+
+Kills all processes possible to be killed.
+
+#### `kill -l 11`
+
+Translates signal number 11 into its signal name.
+
+#### `kill -L`
```

Comment:
`-L` doesn't seem to be a valid flag, but the lowercase version does. Could you also include the output of this command in your documentation?